contentgrid-spring-data-rest/src/testFixtures/java/com/contentgrid/spring/test/fixture/invoicing/model/support/Content.java

```diff
@@ -28,6 +30,7 @@ public class Content {
 
     @MimeType
     @CollectionFilterParam(value = "mimetype")
+    @Pattern(regexp = "^(application|audio|font|image|model|text|video)/[a-z0-9!#$&\\-^_.+]+$", flags = Flag.CASE_INSENSITIVE)
```

Comment:
According to StackOverflow you're missing [a few possible types](https://stackoverflow.com/questions/25201083/regex-to-match-and-validate-internet-media-type/48046041#48046041) but I'm not sure we need to do it this strictly anyway.

Reply:
Oh, I know.
This is just a test regex pattern, not the final one that may actually be generated and used in applications.